### ISSUE TITLE
lexicon_text advance cache key bug

### ISSUE THREAD

**Issue by ringotf** [tabularelf/lexicon#15]
Hi, great project, really useful... I think I found a cache bug in lexicon_text where it doesn't generate the correct cache key when you have LEXICON_CACHE_ARG_THRESHOLD (or more arguments). 
It includes _textEntry in the cache key twice because it's not iterating the arguments correctly. The below 2 changes seem to work, hope they make sense:

```
for(var _i = 1; _i <= _count; ++_i) {       //change to <=
	if (_i > argument_count) break;
	_args[_i-1] = argument[_i];           //change to argument[_1];
```

https://github.com/tabularelf/lexicon/blob/14cb8bec5c1fecd802021d06a4d990b853cd792a/scripts/lexicon_text/lexicon_text.gml#L47-L49

**Comment by tabularelf** [comment]
I apologize, I have been a bit occupied over the last few months. I'm currently in the middle of making v3, which has a bunch of improvements.

I will apply a patch update to this soon.

**Comment by tabularelf** [comment]
Update: Fixed! 2.1.5 is out! ^^
